src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>net45;netcoreapp2.0</TargetFrameworks>
     <Version>0.2.2-alpha</Version>
     <RootNamespace>Datadog.Trace.ClrProfiler</RootNamespace>
+    <LangVersion>7.3</LangVersion>
```

Comment:
C# 7.3 added delegate generic constraints (`where T : delegate`), which is useful for `DynamicMethodBuilder`.